### PR TITLE
Embed improvements

### PR DESCRIPTION
**Pull request by steve-gr4vy**
* Small wording updates
* Add table for embed parameters

### CODE REVIEW

File: guides/web-checkout/embed.mdx [hunk 1]
```diff
@@ -174,6 +182,100 @@ will heavily depend on the enabled payment services in your account.
...
+        <td>ThemeOptions See <a href="../embed/theme">Theming</a></td>
+    </tr>
+    <tr>
+        <td>onEvent</td>
+        <td><b>Optional.</b> A callback function that's used to subscribe to events</td>
+        <td>(eventName: string, data: any) => void</td>
+    </tr>
+  </tbody>
+</table>
+
```
Comment (cbetta) [inline guides/web-checkout/embed.mdx:278]: Why is this HTML and not Markdown?

File: guides/web-checkout/key.mdx [hunk 2]
```diff
@@ -13,13 +13,12 @@ Web Token** (also called a JWT) which is passed as in a HTTP header.
 
 ```bash
 curl -X GET https://api.{gr4vyId}.gr4vy.app/transactions \
-  -H "authorization: bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIi..."
+  -H "Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIi..."
```
Comment (cbetta) [inline guides/web-checkout/key.mdx:16]: I'd like to default to lower case. Any reason why you capitalized it?

File: guides/web-checkout/embed.mdx (hunk 1 above)
Comment (steve-gr4vy) [reply guides/web-checkout/embed.mdx:278]: Following the Docusaurus docs here: https://docusaurus.io/docs/markdown-features I understood that Docusaurus only supports standard markdown and not extended markdown, maybe I missed it?

Comment (steve-gr4vy) [reply guides/web-checkout/embed.mdx]: Have updated this to a markdown table.

File: guides/web-checkout/key.mdx (hunk 2 above)
Comment (steve-gr4vy) [reply guides/web-checkout/key.mdx]: Have reverted to lowercase. ugg.  :)

File: guides/web-checkout/embed.mdx [hunk 3]
```diff
@@ -92,11 +92,15 @@ token you generated in the previous step.
 import Gr4vyEmbed from (`@gr4vy/embed-react)
 // or: const { default: Gr4vyEmbed } = require(`@gr4vy/embed-react`)
 
+<form action='/' id='payment-form'>
+  <input type="submit" /> 
+</form>
 <Gr4vyEmbed
```
Comment (cbetta) [inline guides/web-checkout/embed.mdx:98]: I'd actually nest the React component in the form.

File: guides/web-checkout/embed.mdx [hunk 4]
```diff
@@ -196,14 +232,16 @@ transaction ID embedded in the form.
 const Gr4vyEmbed = require(`@gr4vy/embed-react`)
 // or: import Gr4vyEmbed from (`@gr4vy/embed-react)
 
-<form action='/' id='checkout' />
+<form action='/' id='payment-form'>
+  <input type="submit" /> 
+</form>
```
Comment (cbetta) [inline guides/web-checkout/embed.mdx:237]: Same here.

**Review by cbetta — APPROVED** [review]
1 small request for change but rest looks great.

File: guides/web-checkout/embed.mdx (hunk 3 above)
Comment (steve-gr4vy) [reply guides/web-checkout/embed.mdx]: done.

**Review by cbetta — APPROVED** [review]

File: guides/web-checkout/embed.mdx [hunk 5]
```diff
@@ -9,8 +9,8 @@ import Image from "../../src/modules/Image";
 import embed from "./assets/embed.jpg";
 
 The final step to is to add [Gr4vy Embed][embed] to your checkout experience.
```
Comment (douglaseggleton) [inline guides/web-checkout/embed.mdx:11]: It says final step, but this is step 3/4. (Inspect transaction is final)

File: guides/web-checkout/index.mdx [hunk 6]
```diff
@@ -12,10 +12,9 @@ get set up with Gr4vy as your web checkout experience.
...
 <Image src={embed} title="The Gr4vy Embed checkout experience" />
 
-1. [We will create an API key][1] needed when generating the signed JWT client token.
-2. [We will set up a server-side SDKs][2] to generate a client token for 
-   authenticating to the Gr4vy API
-   to the APIs.
+1. [We will create an API key][1] which is needed to generate the signed JWT client token in step 2.
+2. [We will set up a server-side SDK][2] to generate a client token for 
+   authenticating to the Gr4vy API.
 3. [And finally we will add **Gr4vy Embed**][3] which will handle the capture of
```
Comment (douglaseggleton) [inline guides/web-checkout/index.mdx:18]: https://deploy-preview-15--gr4vy-docs.netlify.app/web-checkout/inspect ? Should there also be step4 ?

**Pull request merged**